layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+{% include "template-variables" layout_product: true %}
+{% include "product-variables" %}
+
+<html class="layout-root {{ language_flags_mode }} {{ language_names_mode }} {{ language_menu_mode }} {{ site_search_mode }}" lang="{{ page.language_code }}" data-view-state="{{ view_mode }}" data-lingual-mode="{{ site_lingual_mode }}" data-menu-main-state="closed">
+  <head prefix="og: http://ogp.me/ns#">
+    {% include "template-head" layout_product: true %}
+  </head>
+
+  {% capture bottom_content_html %}
+    {% unless editmode %}
+      {% content bind=product name="content" %}
+    {% endunless %}
+  {% endcapture %}
+
+  {% capture bottom_content_size %}
+    {{ bottom_content_html | size | minus: 1 }}
+  {% endcapture %}
+
+  {% unless bottom_content_size contains "-" %}
+    {% assign bottom_content_has_content = true %}
+  {% endunless %}
+
+  <body class="layout-container item-page">
+    {% include "template-svg-spritesheet" %}
+
+    <div class="layout-wrap">
+      {% include "layout-header" %}
+
+      <main class="layout-body layout-body--narrow">
+
+        <div class="content-inner product-content">
+          <div class="layout-body__content">
+            <div class=" items-body">
+              <div class="content-illustrations">
+                {% assign product_image = product.image %}
+
+                {% if product_image != blank %}
+                  {% assign item_image_state = "with-image" %}
+                {% else %}
+                  {% assign item_image_state = "without-image" %}
+                {% endif %}
+
+                <div class="content-item-box {{ item_image_state }} js-content-item-box">
+                  <div class="item-top">
+                    {%- if product_image != blank -%}
+                      <div class="loader js-loader"></div>
+                      <div class="top-inner aspect-ratio-inner product-page">
+                        {%- assign image_class = "item-image not-cropped" -%}
+                        {% image product_image target_width: "1280" class: image_class loading: "lazy" %}
```

Comment:
Here seems that here is something mixed – JS Lazy Load and IMG `loading="lazy"` attribute.

`<div class="loader js-loader"></div>` should be removed or

image tag must be replaced with `image_data` to use 

```liquid
{%- assign image_class = "item-image js-lazyload not-cropped" -%}
{% image_data productImage target_width="1280" class: imageClass %}
```